src/meta/processors/parts/CreateSpaceProcessor.cpp

```diff
@@ -268,16 +268,12 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
   LOG(INFO) << "Create space " << spaceName;
 }
 
-StatusOr<Hosts> CreateSpaceProcessor::pickHostsWithZone(
+ErrorOr<nebula::cpp2::ErrorCode, Hosts> CreateSpaceProcessor::pickHostsWithZone(
     const std::vector<std::string>& zones,
     const std::unordered_map<std::string, Hosts>& zoneHosts) {
   Hosts pickedHosts;
   nebula::cpp2::ErrorCode code = nebula::cpp2::ErrorCode::SUCCEEDED;
   for (auto iter = zoneHosts.begin(); iter != zoneHosts.end(); iter++) {
-    if (code != nebula::cpp2::ErrorCode::SUCCEEDED) {
-      break;
-    }
-
     if (iter->second.empty()) {
       LOG(ERROR) << "Zone " << iter->first << " is empty";
       code = nebula::cpp2::ErrorCode::E_INVALID_PARM;
```

Comment:
how about this error? Should it have a specific error code?

Reply:
ZONE IS EMPTY is OK ?

Reply:
good to me